SerialPrograms/Source/PokemonSV/Inference/Overworld/PokemonSV_NoMinimapDetector.cpp

```diff
@@ -0,0 +1,44 @@
+/*  No Minimap Detector
+ *
+ *  From: https://github.com/PokemonAutomation/Arduino-Source
+ *
+ */
+
+#include "Kernels/Waterfill/Kernels_Waterfill_Session.h"
+#include "CommonFramework/Globals.h"
+#include "CommonFramework/VideoPipeline/VideoOverlayScopes.h"
+#include "CommonFramework/ImageMatch/ImageDiff.h"
+#include "CommonFramework/ImageMatch/ExactImageMatcher.h"
+#include "CommonFramework/ImageTools/BinaryImage_FilterRgb32.h"
+#include "CommonFramework/ImageTools/WaterfillUtilities.h"
+#include "CommonFramework/ImageMatch/WaterfillTemplateMatcher.h"
+#include "PokemonSV_NoMinimapDetector.h"
+
+//#include <iostream>
+//using std::cout;
+//using std::endl;
+
+namespace PokemonAutomation{
+namespace NintendoSwitch{
+namespace PokemonSV{
+
+
+
+NoMinimapDetector::NoMinimapDetector(Logger& logger, Color color)
+    : m_color(color)
```

Comment:
Why do we need `m_color` and `m_ball` separately just for the overlay? Can't we make `m_overworld` do the overlay directly?

Reply:
I wanted NoMinimapDetector's overlay to look slightly different than the Overworld detector's overlay. Overworld detector has 3 boxes in its overlay, while NoMinimapDetector's overlay is only the smallest inner box. This way, users can tell when we're looking or the Overworld, and when we're looking for no overworld.

I don't feel strongly about this though. If you prefer to use the Overworld's overlay, I can change it.